src/Fortran-interface/bml_introspection_m.F90

```diff
@@ -93,6 +93,10 @@ function bml_get_type(a)
       bml_get_type = "dense"
     case(2)
       bml_get_type = "ellpack"
+    case(3)
+      bml_get_type = "ellblock"
```

Comment:
I see a fix for ellblock in here. I had missed that one. What was the consequence of not having it?